stage6/04-gnuradio-m2k/00-run.sh

```diff
@@ -12,9 +12,10 @@ QWT_BRANCH=qwt-6.1-multiaxes
 QWTPOLAR_BRANCH=master
 LIBSIGROK_BRANCH=master
 LIBSIGROKDECODE_BRANCH=master
+SCOPY_BRANCH=master
 
 ARCH=arm
-JOBS=-j4
+JOBS=-j12
```

Comment:
maybe leave this default as 4, and parametrize it with 
```
NUM_JOBS=${NUM_JOBS:-4}
JOBS=-j${NUM_JOBS}
```

then provide 12 on systems that can handle this;
otherwise if there are 1 or 2 cores, 12 is really too much

Reply:
i was thinking of our build server. it is true that is should parametrize this